packages/next/client/page-loader.js

```diff
@@ -78,17 +78,16 @@ export default class PageLoader {
       this.loadingRoutes[initialPage] = true
     }
 
-    if (process.env.NODE_ENV === 'production') {
```

Comment:
Why was this if clause removed?

Reply:
The rewrites are added to the buildManifest and used in both dev and production mode so this updates it to load the buildManifest in dev mode also